README.md

```diff
@@ -0,0 +1,27 @@
+## Problem Description 
```

Comment:
I would prepend a brief summary to describe this repository's content itself, e.g.:

```
An addon that measures DNSSEC interference by middleboxes.
```

Reply:
Addressed by 1edb62a